node/tests/AsyncClient.test.ts

```diff
@@ -12,8 +12,6 @@ const FreePort = require("find-free-port");
 
 const PORT_NUMBER = 4000;
 
-type EmptyObject = Record<string, never>;
```

Comment:
Why remove this?

Reply:
`Context` never used in shared tests, so I removed all references.